test/plugin.test.ts

```diff
@@ -9,7 +9,7 @@ import * as httpserver from "./httpserver"
 const fsp = fs.promises
 
 // Jest overrides `require` so our usual override doesn't work.
-jest.mock("code-server", () => codeServer)
+jest.mock("code-server", () => codeServer, { virtual: true })
```

Comment:
> virtual mocks – mocks of modules that don't exist anywhere in the system

🤯 TIL what a `virtual` mock is

[Source](https://jest-bot.github.io/jest/docs/jest-object.html#jestmockmodulename-factory-options)

Reply:
Haha yeah idk why it was passing locally without this for me though. :shrug: